pkg/cli/containers_test.go

```diff
@@ -38,36 +87,40 @@ func TestContainer(t *testing.T) {
 				Output: "",
 			},
 			commandContext: CommandContext{
-				ClientFactory: &testdata.MockClientFactory{},
-				StdOut:        w,
-				StdErr:        w,
-				StdIn:         strings.NewReader("y\n"),
+				ClientFactory: &testdata.MockClientFactoryManual{
+					Client: mClient,
+				},
+				StdOut: w,
+				StdErr: w,
+				StdIn:  strings.NewReader("y\n"),
 			},
 			args: args{
 				args:   []string{},
-				client: &testdata.MockClient{},
+				client: mClient,
 			},
 			wantErr: false,
-			wantOut: "NAME              APP       IMAGE     STATE     RESTARTCOUNT   CREATED    MESSAGE\nfound.container                                 0              292y ago   \n",
+			wantOut: "NAME               APP       IMAGE     STATE     RESTARTCOUNT   CREATED    MESSAGE\nfound.container1                                 0              292y ago   \nfound.container2                                 0              292y ago   \n",
 		},
 		{
-			name: "acorn container found.container", fields: fields{
+			name: "acorn container found.container1", fields: fields{
 				All:    false,
 				Quiet:  false,
 				Output: "",
 			},
 			commandContext: CommandContext{
-				ClientFactory: &testdata.MockClientFactory{},
-				StdOut:        w,
-				StdErr:        w,
-				StdIn:         strings.NewReader("y\n"),
+				ClientFactory: &testdata.MockClientFactoryManual{
+					Client: mClient,
+				},
+				StdOut: w,
+				StdErr: w,
+				StdIn:  strings.NewReader("y\n"),
 			},
 			args: args{
-				args:   []string{"--", "found.container"},
-				client: &testdata.MockClient{},
+				args:   []string{"--", "found.container1"},
+				client: mClient,
 			},
 			wantErr: false,
-			wantOut: "NAME              APP       IMAGE     STATE     RESTARTCOUNT   CREATED    MESSAGE\nfound.container                                 0              292y ago   \n",
+			wantOut: "NAME               APP       IMAGE     STATE     RESTARTCOUNT   CREATED    MESSAGE\nfound.container1                                 0              292y ago   \n",
```

Comment:
I realize you didn't do this originally, but there is a problem here. These tests are going to start failing every year because `292y` will increase to `293y`, and so on.

While you are here changing this code, could you add a `CreationTimestamp` to the `ObjectMeta` objects here? Something like `time.Now().AddDate(-10, 0, 0)` should make it consistent for the foreseeable future.